notes/1.8/notes.MD

```diff
@@ -0,0 +1,4 @@
+The good about this book is that it uncovers a whole area of JS that's mostly underutilized with practical examples and comparisons with other languages so that it makes it engaging
```

Comment:
These notes are very limited; please include information about all relevant topics and concepts reviewed while reading this book. (Concurrency, parallelism, threads, processes, web assembly, etc.)